libs/blocks/merch-cards/merch-cards.css

```diff
@@ -0,0 +1,7 @@
+.section[class*="-merch-cards"] .full-width {
+  grid-column: 1 / -1;
+}
+
+merch-cards {
+  min-height: 200px;
```

Comment:
where is that 200px come from?

Reply:
@npeltier it is an arbitrary value to allocate some space for the cards.
in a second PR,, I'll adjust spacing after feedback from different stakeholders and more visual tests.